code/modules/clothing/outfits/ert.dm

```diff
@@ -171,6 +179,8 @@
 	belt = /obj/item/gun/energy/e_gun
```

Comment:
They already get a firearm, they don't need a second one.

Reply:
Idea is that you would have a second one as a backup. In case of being stunned down or such. All in all I would love to flat out replace that gun with the mini-e gun but that would be a massive nerf to a antag